server/infrastructure/portal/mapper/PortalDataMapper.ts

```
@@ -0,0 +1,150 @@
+import { StudentStatusType } from '@/server/domain/student/models/AcademicInfo';
+import {
+  RawPortalData,
+  RawPortalGradeResponseDTO,
+  RawPortalSemesterDTO,
+  RawPortalStudentDTO,
+} from '../dto/portal-types';
+
+export class PortalDataMapper {
+  static toPortalData(raw: RawPortalData): PortalData {
+    return {
+      student: this.toPortalStudentInfo(raw.student),
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)